src/building.md

```diff
@@ -239,3 +239,99 @@ git clone https://github.com/spack/spack.git
 cd spack
 ./bin/spack install -v mfem
 ```
+
+## Building MFEM with CMake
+To build a serial version of MFEM with CMake first create a build directory. For example, using a build directory named `build`:
+```sh 
+mkdir build 
+cd build 
+```
+
+Run the CMake configuration on the MFEM source directory.
+```sh 
+cmake ..
+```
+
+Run the build command associated with the CMake configuration, specifying the number of parallel build tasks with the `-j` flag (4 tasks in this case). 
+```sh 
+cmake --build . -j 4
+```
+### Parallel build using CMake
+To build a parallel version of MFEM with CMake first build METIS and Hypre as described above.
+From the MFEM source directory, create a build directory. For example, using a build directory named `build`:
+```sh 
+cd mfem-4.5
+mkdir build 
+cd build 
+```
+
+Run the CMake configuration on the MFEM source directory using the `MFEM_USE_MPI` CMake variable to enable MPI. 
+This will automatically search for the system MPI implementation, the METIS installation (in `<mfem-source-dir>/../metis-4.0`), and Hypre installation (in `<mfem-source-dir/../hypre`).
+```sh 
+cmake .. -DMFEM_USE_MPI=YES
+```
+Run the build command associated with the cmake configuration, specifying the number of parallel build tasks with the `-j` flag (4 tasks in this case). 
```

Comment:
```suggestion
Alternatively, the `MFEM_FETCH_TPLS` CMake variable can be used to enable fetching of both METIS and hypre, which will then be built alongside MFEM.
```sh
cmake .. -DMFEM_USE_MPI=YES -DMFEM_FETCH_TPLS=YES
```
Now, run the build command associated with the cmake configuration, specifying the number of parallel build tasks with the `-j` flag (4 tasks in this case). 
```

I would make the argument that using `MFEM_FETCH_TPLS` should be the default option in this documentation to avoid the need for discussing paths to Hypre and MFEM

Reply:
Sounds good!
I also recently fixed my computer with an AMD gpu so I can take another pass at the gpu instructions to get them up to date afterwards

Reply:
Excellent, I commit the change with my suggestion... again, feel free to revert or iterate on it as needed.

I will note this PR is being considered for the upcoming MFEM v4.9 release, which will require that this PR be ready to merge by early next week.